pgcli/main.py

```diff
@@ -520,8 +522,18 @@ def cli(database, user, host, port, prompt_passwd, never_prompt, dbname,
             '\thost: %r'
             '\tport: %r', database, user, host, port)
 
+    obscure_process_password()
     pgcli.run_cli()
 
+def obscure_process_password():
```

Comment:
I prefer `obfuscate_process_password` over `obscure_process_password` since it is hard to discern whether obscure is used as a noun or a verb at first glance.